### DISCUSSION TITLE
Check code compatibility including dependencies

### DISCUSSION THREAD

**Discussion by ego-lay-atman-bay** [netromdk/vermin#248] [Q&A] [answered]
I just recently used vermin to check what the minimum python version my module can run on, and I got 3.8. I set up a 3.8 venv to verify that it could indeed run, and then when I installed the dependencies (requirements), it through an error saying that numpy didn't have a compatible version.

I'm wondering how I can use vermin to check the minimum version my code can run in, as well as external modules.

**Comment by netromdk** [comment]
Vermin can only check code. So either you check the code of the dependencies or you have to look up what each project states they support of versions. The latter is, unfortunately, out-of-scope of the project.
